### PR TITLE
Add Prague Containers meetup

### PR DESCRIPTION
**Pull request by mattj-io**
## Description
https://jira.mesosphere.com/browse/DCOS_COMMUNITY-196

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/events.json [hunk 1]
```diff
@@ -541,7 +541,7 @@
   { "title": "Mesos and DC/OS Intro (Meetup)",
       "image": "/assets/images/events/Prague_2.jpg",
```
Comment (pleia2) [inline src/events.json]: Since there is a photo from the event, if we have the rights to it you could replace this one :)

File: src/events.json [hunk 2]
```diff
@@ -541,7 +541,7 @@
...
       "date": "2017-11-28",
-      "url": "https://www.meetup.com/Prague-Containers-Meetup/events/244571013/?a=socialmedia"
+      "url": "/events/2017/mesos-and-dc-os-in-Prague/"
```
Comment (pleia2) [inline src/events.json]: Throws a 404, the URL is actually: /events/2017/mesos-and-dc-os-in-prague/

File: src/events/2017-11-28_prague_containers_meetup.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Mesos and DC/OS in Prague
+date: 2017-11-28
+speaker: Matt Jarvis
+eventimage: Prague_2.jpg
```
Comment (pleia2) [inline src/events/2017-11-28_prague_containers_meetup.md]: Same comment as above WRT image.

**Comment by mattj-io** [comment]
Changes as requested

File: src/events.json (hunk 1 above)
Comment (pleia2) [reply src/events.json]: Almost there, we want to update this image to be prague_containers.jpg as well.

**Pull request merged**